astroquery/utils/commons.py

```diff
@@ -470,22 +470,3 @@ def parse_votable(content):
     """
     tables = votable.parse(six.BytesIO(content), pedantic=False)
     return tables
-
-
-def _open_shelve(shelffn, withclosing=False):
```

Comment:
We used to support very old astropy versions, but 3.1 is now fairly new so I agree that we can remove this now.